views/dashboard.ejs

```diff
@@ -0,0 +1,293 @@
+<head>
+  <title>Rose City Resource Admin Page</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.css" />
+  <script>
+
```

Comment:
Might be good to separate out this js your html template for better readability in this file.